satellite/console/consoleweb/server.go

```diff
@@ -72,7 +75,8 @@ type Server struct {
 	listener net.Listener
 	server   http.Server
 
-	schema graphql.Schema
+	schema    graphql.Schema
+	templates map[string]*template.Template
```

Comment:
Take a look at https://github.com/storj/storj/blob/master/satellite/marketingweb/server.go#L39

Reply:
In other words you can make a struct with named members and invoke those.